content/en/smart-contracts/fundamentals/directdealmaking

```diff
@@ -0,0 +1,47 @@
+---
+title: "Direct Dealmaking"
+description: "This article conceptually explains the client contract / direct deal-making process."
+lead: "This article conceptually explains the client contract / direct deal-making process."
+draft: false
+images: []
+type: docs
+menu:
+  smart-contracts:
+    parent: "smart-contracts-fundamentals"
+    identifier: "direct-dealmaking"
+weight: 110
+toc: true
+aliases:
+
+---
+
+## Direct Dealmaking
+
+The design of Filecoin is to allow clients to store large amounts of data for a long period of time.  In order to make a deal proposal via smart contract on the FVM directly with a storage provider for data > 4GB, the [client contract](github.com/filecoin-project/fvm-starter-kit-deal-making/blob/main/contracts/DealClient.sol) can be used. 
```

Comment:
Please remove the fvm starter kit link and link to tutorial instead (https://docs.filecoin.io/smart-contracts/developing-contracts/client-contract-tutorial/) for more consistency. Any edit to the client contract flow will be updated on that doc